src/browser/modules/Editor/ActionButtons.tsx

```diff
@@ -34,7 +34,10 @@ interface ActionButton {
   iconColor?: string
 }
 
-const ActionButtons = ({ buttons, width = 24 }: ActionButtonProps) => (
```

Comment:
This component is hit by the rule of that all exported things need to be fully typed, so I added a return type back